res/core/messages.xml

```diff
@@ -8441,4 +8441,15 @@
     <text locale="en">"$unit($unit) in $region($region): '$order($command)' - Heroes cannot recruit."</text>
   </message>
 
+  <message name="dragon_growl" section="mail">
+    <type>
+      <arg name="dragon" type="unit"/>
+      <arg name="number" type="int"/>
+      <arg name="target" type="region"/>
+      <arg name="choice" type="int"/>
+    </type>
+    <text locale="de">"$unit($dragon): \"$if($eq($choice,0), "Groaaaam... ", "")$if($eq($choice,1), "Chrrrr... ", "")$if($eq($choice,2), "Roooarrr... ", "")$if($eq($choice,3), "Tschrrrk... ", "")$if($eq($choice,4), "Shhhhhh... ", "")$if($eq($number,1), "Ich rieche", "Wir riechen") etwas in $region($target)\"."</text>
```

Comment:
So viel Logik in den Text zu bauen (die ganzen $if checks) finde ich inzwischen eher unschoen. Problem ist natuerlich die Ueberzetzung, wenn es uns daran liegt, "Groaaaam" zu internationalisieren? Idee dazu:
<arg name="growl" type="string">
<text locale="de">"Der Drache sagt: $t($growl)"</text>
<text locale="en">"The dragon says: $t($growl)"</text>
Wobei $t eine Funktion ist, die den uebersetzten String liefert fuer ein token, das in $growl steht. in strings.xml fuegen wir dann growl_1, growl_2, growl_3, usw. hinzu.
Ich habe im Urin, dass es so etwas wie $t schon gibt, kann das aber gerade nicht finden.